### PR TITLE
Remove Sha256Checksum parameter of example of New-AzImageBuilderCustomizerObject

### LINKED ISSUE #12200: example of New-AzImageBuilderCustomizerObject needs to remove Sha256Checksum parameter
## Description

`New-AzImageBuilderCustomizerObject’ cmdlet requires to use a Sha256 checksum however this is not possible in the case of an inline script. We need to docs as well.

## Steps to reproduce

```powershell

```

## Environment data

```

```

## Module versions

```powershell

```

## Debug output

```

```

## Error output

```

```

### PR DESCRIPTION
**Pull request by LucasYao93**
## Description

Fix #12200

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/ImageBuilder/custom/Az.ImageBuilder.custom.psm1 [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # region Generated 
   # Load the private module dll
-  $null = Import-Module -PassThru -Name (Join-Path $PSScriptRoot '..\bin\Az.ImageBuilder.private.dll')
+  $null = Import-Module -PassThru -Name (Join-Path $PSScriptRoot '../bin/Az.ImageBuilder.private.dll')
```
Comment (dingmeng-xue) [inline src/ImageBuilder/custom/Az.ImageBuilder.custom.psm1:3]: I don't think this file should be changed.

File: src/ImageBuilder/custom/readme.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Custom
-This directory contains custom implementation for non-generated cmdlets for the `Az.ImageBuilder` module. Both scripts (`.ps1`) and C# files (`.cs`) can be implemented here. They will be used during the build process in `build-module.ps1`, and create cmdlets into the `..\exports` folder. The only generated file into this folder is the `Az.ImageBuilder.custom.psm1`. This file should not be modified.
+This directory contains custom implementation for non-generated cmdlets for the `Az.ImageBuilder` module. Both scripts (`.ps1`) and C# files (`.cs`) can be implemented here. They will be used during the build process in `build-module.ps1`, and create cmdlets into the `../exports` folder. The only generated file into this folder is the `Az.ImageBuilder.custom.psm1`. This file should not be modified.
```
Comment (dingmeng-xue) [inline src/ImageBuilder/custom/readme.md:2]: this file should not be changed.

File: src/ImageBuilder/docs/Az.ImageBuilder.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 Module Name: Az.ImageBuilder
-Module Guid: bdedc683-d9b6-41ea-b310-d068b8c72305
+Module Guid: e90b2588-5031-43ea-81b5-9a419d2fec07
```
Comment (dingmeng-xue) [inline src/ImageBuilder/docs/Az.ImageBuilder.md:3]: Should be changed?

File: src/ImageBuilder/docs/readme.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # Docs
-This directory contains the documentation of the cmdlets for the `Az.ImageBuilder` module. To run documentation generation, use the `generate-help.ps1` script at the root module folder. Files in this folder will *always be overriden on regeneration*. To update documentation examples, please use the `..\examples` folder.
+This directory contains the documentation of the cmdlets for the `Az.ImageBuilder` module. To run documentation generation, use the `generate-help.ps1` script at the root module folder. Files in this folder will *always be overriden on regeneration*. To update documentation examples, please use the `../examples` folder.
```
Comment (dingmeng-xue) [inline src/ImageBuilder/docs/readme.md:2]: his file should not be changed.

File: src/ImageBuilder/docs/Az.ImageBuilder.md (hunk 3 above)
Comment (wyunchi-ms) [reply src/ImageBuilder/docs/Az.ImageBuilder.md:3]: Agree

**Comment by LucasYao93** [comment]
I will open a new PR to fix this issue. @dingmeng-xue & @wyunchi-ms

**Pull request closed without merging**